cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -247,11 +247,12 @@ friendlyTxBodyImpl
               ++ ( monoidForEraInEon @ConwayEraOnwards
                     era
                     ( \cOnwards ->
-                        case txProposalProcedures of
-                          Nothing -> []
-                          Just (Featured _ TxProposalProceduresNone) -> []
-                          Just (Featured _ (TxProposalProcedures lProposals _witnesses)) ->
-                            ["governance actions" .= (friendlyLedgerProposals cOnwards $ toList lProposals)]
+                        conwayEraOnwardsConstraints cOnwards $
+                          case txProposalProcedures of
+                            Nothing -> []
+                            Just (Featured _ pp) -> do
+                              let lProposals = toList $ convProposalProcedures pp
+                              ["governance actions" .= (friendlyLedgerProposals cOnwards lProposals)]
```

Comment:
Previously, in the case `TxProposalProceduresNone`, the output JSON would not contain the key `governance actions`.

Not we will output `"governance actions" = []`. Is this intended?

Reply:
Good catch. That shouldn't happen. I'll fix this.